GenderPayGap.Core/Helpers/CsvWriter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
+using CsvHelper.Configuration;
+using CsvHelper.TypeConversion;
+
+namespace GenderPayGap.Core.Helpers
+{
+    public static class CsvWriter
+    {
+
+        public static T Write<T>(Func<MemoryStream, StreamReader, StreamWriter, CsvHelper.CsvWriter, T> write)
+        {
+            var config = new CsvConfiguration(CultureInfo.CurrentCulture)
+            {
+                InjectionCharacters = InjectionCharacters, SanitizeForInjection = true
+            };
+            using (var memoryStream = new MemoryStream())
+            using (var streamReader = new StreamReader(memoryStream))
+            using (var streamWriter = new StreamWriter(memoryStream))
+            using (var csvWriter = new CsvHelper.CsvWriter(streamWriter, config))
+            {
+                csvWriter.Configuration.TypeConverterCache.AddConverter<string>(new CustomConverter());
+                return write(memoryStream, streamReader, streamWriter, csvWriter);
+            }
+        }
+
+        /*
+         * The default injection characters are: '=', '@', '+', '-'
+         * We ignore '-' because we don't want the negative figures to be sanitized (i.e., prepended with a tab character)
+         * We sanitize manually the other fields that start with '-' using the CustomConverter
+         */
+        private static readonly char[] InjectionCharacters = {'=', '@', '+'};
```

Comment:
Should this be at the top (just because the Write function uses it)? I understand maybe the convention is different?
Fine if you don't agree, I've approved the changes 😁.
(sorry I'm only suggesting this now)

Reply:
@Lucas-Fernandes-Nogueira
I usually order the items by their access:
- public
- protected
- private

However, as I haven't written that much C# code, I've googled that and found [this Stackoverflow answer](https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods)(the first answer) that I find useful. I will update this class accordingly. Thanks for flagging!

Reply:
Thanks for sharing that link, it's really helpful!